src/config.py

```diff
@@ -24,7 +24,7 @@
 
 # User specific constants.
 COUNTRY = core.Country.GERMANY
-TAX_YEAR = 2020
+TAX_YEAR = 2021
```

Comment:
leave it at 2020

Reply:
Yep, wasn't intended, it was just meant to be set locally for testing. Will change it back.